libplatsupport/src/mach/bcm/pl011_uart.c

```diff
@@ -96,16 +98,18 @@ static inline void pl011_uart_disable_fifo(ps_chardevice_t *dev)
     r->lcrh &= ~LCRH_FEN;
 }
 
-static inline void pl011_uart_enable_rx_irq(ps_chardevice_t *dev)
+static inline void pl011_uart_enable_irqs(ps_chardevice_t *dev)
```

Comment:
Now that we enable multiple interrupt, can you add a comment here what each enabled interrupt reports. The acronyms are not intuitive.

Reply:
done